library/cartridge_edit_topology.py

```diff
@@ -277,20 +293,27 @@ def get_edit_replicasets_params(replicasets, cluster_replicasets, cluster_instan
     return edit_replicasets_params, None
 
 
-def get_edit_servers_params(instances_to_expel, cluster_instances):
-    edit_servers_params = [
-        {
+def get_expel_servers_params(instances, cluster_instances):
+    expel_servers_params = []
+    for instance_name, instance_params in instances.items():
+        if instance_params.get('expelled') is not True:
+            continue
+
+        if instance_name not in cluster_instances:
+            continue
+
+        if not cluster_instances[instance_name].get('uuid'):
+            continue
+
+        expel_servers_params.append({
             'uuid': cluster_instances[instance_name]['uuid'],
             'expelled': True,
-        }
-        for instance_name in instances_to_expel
-        if instance_name in cluster_instances and cluster_instances[instance_name].get('uuid') is not None
-    ]
+        })
 
-    return edit_servers_params
+    return expel_servers_params, None
```

Comment:
There are no expressions like `return None, err`. Let's do:
```suggestion
    return expel_servers_params
```